pandas/core/indexes/range.py

```diff
@@ -49,12 +49,6 @@ class RangeIndex(Int64Index):
     copy : bool, default False
         Unused, accepted for homogeneity with other index types.
 
-    Attributes
-    ----------
-    start
-    stop
-    step
```

Comment:
why remove those?

Reply:
those are not actual attributes for what I saw, I'll revert here anyway, and let's take care of `RangeIndex` in a separate PR

Reply:
They recently got added as actual attributes, IIRC

Reply:
you're right, I think I didn't test in master